requirements.txt

```diff
@@ -1,6 +1,6 @@
 networkx>=2.0
 numpy>=1.16.3
 scipy>=1.0.0
-tensorflow>=1.3.0,<1.7
+tensorflow==1.3
```

Comment:
By pinning this to 1.3, we can ensure that everyone on the simulator is now using the same TF version.